src/applications/ezr/tests/unit/components/IntroductionPage/SaveInProgressInfo.unit.spec.js

```diff
@@ -183,6 +185,51 @@ describe('ezr <SaveInProgressInfo>', () => {
           expect(selector).to.contain.text(content['alert-server-title']);
         });
       });
+
+      context('when the user has a preferred facility on file', () => {
+        it('should not render preferred facility alert', () => {
+          const { props, mockStore } = getData({
+            showLoader: false,
+            loggedIn: true,
+            loaState: 3,
```

Comment:
Just for my clarity is this intentionally called 'loaState'? Or could this be a typo for 'loadState'?

Reply:
LOA is “level of assurance” and it relates to the verification of the user account. So if you see a user that has an LOA status of 1, that means they have created an account, but they have not verified their identity. This restricts most of the functions you can perform as a logged in user (such as filling out our forms and other forms, editing personal info, etc). Users with an LOA status of 3 have fully verified their identity and are authorized to fill out forms and update their profile data.